e2e/wdio/platform/pages/button.po.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForElDisplayed } from '../../driver/wdio';
+
+export class ButtonPo extends BaseComponentPo {
+    static checkRtlSwitch() {
+        throw new Error('Method not implemented.');
+    }
+    static saveExampleBaselineScreenshot() {
+        throw new Error('Method not implemented.');
+    }
+    static compareWithBaseline(): any {
+        throw new Error('Method not implemented.');
+    }
+    private url = '/menu-button';
+    root = '#page-content';
+
+    typeButtons = 'fdp-button-types-example fdp-button';
+    sizeButtons = 'fdp-button-sizes-example fdp-button';
+    iconButtons = 'fdp-button-icons-example fdp-button';
+    stateButton = 'fdp-button-state-example fdp-button:nth-child(1)';
```

Comment:
:nth-child(1) part is not needed. 1st button automatically chosen by methods unless index provided.